package.json

```diff
@@ -51,6 +51,7 @@
     "webpack": "webpack",
     "dev": "babel src --out-dir ${WEBAPP_DIR:-../mattermost-webapp}/node_modules/mattermost-redux --source-maps",
     "dev:watch": "babel --watch src --out-dir ${WEBAPP_DIR:-../mattermost-webapp}/node_modules/mattermost-redux --source-maps",
+    "dev-mobile:watch": "babel --watch src --out-dir ${WEBAPP_DIR:-../mattermost-mobile}/node_modules/mattermost-redux --source-maps",
```

Comment:
can we make this `MOBILE_DIR` | `RN_DIR` ? I'm 0/5 on the name but it should definitely be different than `WEBAPP_DIR`

Reply:
right. I copy/pasted and didn't realize that it shouldn't use the same var. Good catch